web/src/components/Article/components/FullLayout/FullLayout.tsx

```diff
@@ -22,7 +23,8 @@ interface Props {
 const FullLayout = ({ article }: Props) => {
   const { width } = useWindowDimensions()
   const isMobile = width < 428
-  console.log('ismobile', isMobile)
+
+  const [english, setEnglish] = React.useState<boolean>(false)
```

Comment:
Would it be an idea to global preference setting that you can configure in your account settings which is then stored into a cookie?

Reply:
Yes, that would be possible indeed
But for now i'm going to merge it like this, pack my bags, and than maybe after do that